test/test.js

```diff
@@ -0,0 +1,25 @@
+const assert = require('assert');
+const got = require('got');
+const cur = require('./../src/index.js');
```

Comment:
1. The first dot is not required if you use two dots right after it (`./../` -> `../`);
2. For API tests, it's better to allow Node.js to load your app from the entrypoint defined in `package.json`. To do it, we can just specify the directory where it is located. In this case it is `../`.